app/logic/users.py

```diff
@@ -47,6 +49,19 @@ def removeUserInterest(program_id, username):
         interestToDelete.delete_instance()
     return True
 
+def getBannedUsers(program):
+    """
+    This function returns users banned from a program.
+    """
+    return ProgramBan.select().where(ProgramBan.program == program)
```

Comment:
This query does not check if they have been unbanned.

Reply:
Fixed